File: my_easy/771/771.py
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        my_set = {c for c in jewels}
        count = 0
        for c in stones:
            if c in my_set:
                count += 1
        return count


s = Solution()
print(f"Solution: {s.numJewelsInStones("aA", "aAAbbbb")}")
print(f"Solution: {s.numJewelsInStones("z", "ZZ")}")

File: my_easy/771/771.js
/**
 * @param {string} jewels
 * @param {string} stones
 * @return {number}
 */
var numJewelsInStones = function (jewels, stones) {
  let set = new Set();
  for (c of jewels) set.add(c);
  let count = 0;
  for (c of stones) if (set.has(c)) count++;
  return count;
};

console.log(`Solution: ${numJewelsInStones("aA", "aAAbbbb")}`);
console.log(`Solution: ${numJewelsInStones("z", "ZZ")}`);
